recipes/harfbuzz/all/conanfile.py

```diff
@@ -141,7 +141,7 @@ def source(self):
                   destination=self.source_folder, strip_root=True)
 
     def build_requirements(self):
-        self.tool_requires("meson/0.63.3")
+        self.tool_requires("meson/0.64.1")
         if not self.conf.get("tools.gnu:pkg_config", default=False, check_type=str):
             self.tool_requires("pkgconf/1.9.3")
 
```

Comment:
```suggestion
        self.tool_requires("glib/2.75.0")

```
harfbuzz invokes glib's `mkenums` executable as part of the build
this probably fixes https://github.com/conan-io/conan-center-index/issues/8023 and https://github.com/conan-io/conan-center-index/issues/14413#issuecomment-1327638642